### ISSUE TITLE
Specifying "element" option changes src attribute instead of href

### ISSUE THREAD

**Issue by wwselleck** [ejci/favico.js#82]
I need to change multiple <link> tags at the same time for browser compatibility reasons, so I'm creating 2 instances and specifying the "element" option on each. But when you specify that option it changes the src attribute instead of the href and the browsers just take the href and ignore the src. 

I see this
```
if (_opt.element) {
	_opt.element.setAttribute('src', url);
}
```

in `link.setIcon`, but am a little confused to its purpose considering the website describes the element option as 
>DOM element where to change "href" attribute (useful in case of multiple link icon elements)

**Comment by ejci** [comment]
You are right. It should be something like this:
```javascript
if (_opt.element) {
    _opt.element.setAttribute('src', url);
    _opt.element.setAttribute('href', url);
}
```

**Comment by ejci** [comment]
Thanks!

**Comment by adam-lynch** [comment]
Should be closed as #83 fixed this

**Comment by ejci** [comment]
yep. Thanks for notifying me. Closed
